src/WORKSPACE

```diff
@@ -59,32 +62,9 @@ http_archive(
 git_repository(
     name = "io_bazel_rules_go",
     remote = "https://github.com/bazelbuild/rules_go.git",
-    # we need rules_go of tag 0.12.0
```

Comment:
Please keep the comment about usage of source_relative.

Reply:
Done.

Reply:
It turns out that the `paths=source_relative` option does not work with the `go_proto_library` provided by io_bazel_rules_go. See: https://github.com/bazelbuild/rules_go/issues/1842

Would it be OK to not use source_relative and so remove this comment? Are there any dependencies of `opencensus-proto` we need to make sure build correctly without it?

Reply:
/cc @yancl Does go_proto_library with absolute path work with your build? If so then LGTM.

Reply:
LGTM:)